### PR TITLE
ocm cut release 2.4 work

### PR DESCRIPTION
**Pull request by zhiweiyin318**

### CODE REVIEW

File: ci-operator/config/open-cluster-management/work/open-cluster-management-work-main.yaml [hunk 1]
```diff
@@ -29,55 +28,6 @@ tag_specification:
...
-- as: sonar-pre-submit
-  commands: |
-    export SELF="make -f /opt/build-harness/Makefile.prow"
-    make -f /opt/build-harness/Makefile.prow sonar/go/prow
-  container:
-    from: src
-  secrets:
-  - mount_path: /etc/sonarcloud/
-    name: acm-sonarcloud-token
 - as: sonar-post-submit
```
Comment (qiujian16) [inline ci-operator/config/open-cluster-management/work/open-cluster-management-work-main.yaml:31]: no sonar either

File: ci-operator/config/open-cluster-management/work/open-cluster-management-work-release-2.4.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+- as: sonar-pre-submit
+  commands: |
+    export SELF="make -f /opt/build-harness/Makefile.prow"
+    make -f /opt/build-harness/Makefile.prow sonar/go/prow
+  container:
+    from: src
+  secrets:
+  - mount_path: /etc/sonarcloud/
+    name: acm-sonarcloud-token
+- as: sonar-post-submit
```
Comment (qiujian16) [inline ci-operator/config/open-cluster-management/work/open-cluster-management-work-release-2.4.yaml:62]: no sonar

File: ci-operator/config/open-cluster-management/work/open-cluster-management-work-main.yaml (hunk 1 above)
Comment (zhiweiyin318) [reply ci-operator/config/open-cluster-management/work/open-cluster-management-work-main.yaml]: done

File: ci-operator/config/open-cluster-management/work/open-cluster-management-work-release-2.4.yaml (hunk 2 above)
Comment (zhiweiyin318) [reply ci-operator/config/open-cluster-management/work/open-cluster-management-work-release-2.4.yaml]: done

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**